addon/components/sessions-grid.js

```diff
@@ -64,6 +73,22 @@ export default Component.extend({
     this.set('confirmDeleteSessionIds', []);
   },
 
+  didRender() {
+    this._super(...arguments);
+    run(() => {
+      window.addEventListener('scroll', () => {
```

Comment:
This event listener needs to be cleaned up when this component is removed. The best way to do that is probably using `ember-lifeline` which has an example at https://ember-lifeline.github.io/ember-lifeline/docs/add-event-listener

Reply:
👍